src/facade/facade.cc

```diff
@@ -20,12 +20,13 @@ constexpr size_t kSizeConnStats = sizeof(ConnectionStats);
 
 ConnectionStats& ConnectionStats::operator+=(const ConnectionStats& o) {
   // To break this code deliberately if we add/remove a field to this struct.
-  static_assert(kSizeConnStats == 112u);
+  static_assert(kSizeConnStats == 120u);
 
   ADD(read_buf_capacity);
   ADD(dispatch_queue_entries);
   ADD(dispatch_queue_bytes);
   ADD(dispatch_queue_subscriber_bytes);
+  ADD(dispatch_queue_max_len);
```

Comment:
here by ADD you will get the sum of all queue max len, is this what you want?

Reply:
I assumed it's a very upper worst case limit (if the to assume load on threads is evenly distributed)

Reply:
What do we want to gain from this metric? this will tell us if we want to use ADD or MAX